### ISSUE TITLE
LiveView 0.18 not supported

### ISSUE THREAD

**Issue by moogle19** [elixir-desktop/desktop#31]
LiveView 0.18 moved the `sigil_H/2` macro from `Phoenix.LiveView.Helpers` to `Phoenix.Component`.
This is used in the `__using__` macro of `Desktop.Menu`.

**Comment by adamwight** [comment]
Found the same thing, trying to use the development head of `desktop` in the example app.

```
== Compilation error in file lib/todo_app/menu_bar.ex ==
** (CompileError) lib/todo_app/menu_bar.ex:7: cannot import Phoenix.LiveView.Helpers.sigil_H/2 because it is undefined or private
    (elixir 1.13.4) src/elixir_import.erl:91: :elixir_import.calculate/6
    (elixir 1.13.4) src/elixir_import.erl:22: :elixir_import.import/4
    (desktop 1.4.1) expanding macro: Desktop.Menu.__using__/1
    lib/todo_app/menu_bar.ex:7: TodoApp.MenuBar (module)
    (elixir 1.13.4) expanding macro: Kernel.use/1
    lib/todo_app/menu_bar.ex:7: TodoApp.MenuBar (module)
```

**Comment by adamwight** [comment]
It might help to set up two CI jobs, one for the oldest stable versions, and another for the latest.  In case it's helpful, here's [the configuration](https://gitlab.com/adamwight/mediawiki_client_ex/-/blob/main/.gitlab-ci.yml) I use for an unrelated elixir library.

**Comment by bkono** [comment]
FWIW, getting compatible with LiveView 0.18.x (and phx 1.7) is a fairly light lift (if you're not worrying about backwards compatibility). [This](https://github.com/bkono/desktop/compare/main...feat/phx1.7) was all I needed to be able to start using it in a fresh phx.new 1.7 app. The desktop-example didn't [need much](https://github.com/bkono/desktop-example-app/compare/main..feat/phx1.7) more either, though there's a number of warning cleanups and additional changes worth making beyond the minimum.

@dominicletz Is this project still active? If so, do you have any suggestions on how to make these changes backwards compatible? Would be happy to PR something if I can do it without breaking existing codebases.

**Comment by dominicletz** [comment]
Definitely active and in use, the backwards compatible move if a good question...

**Comment by dominicletz** [comment]
This has been fixed 1.4.4 -- Please have a look if that version works for you. It's using a conditional compile check for backwards compilation.

**Comment by bkono** [comment]
Perfect, latest updates worked for me. Thanks for getting that updated.
